features/widgets/form_fields_select.feature

```diff
@@ -23,15 +23,15 @@ Feature: Form Fields: Select
   Scenario: selected option
     Then we can get the text of the selected option with:
       """
-      widget(:my_form).my_select #=> "One"
+      widget(:my_form).my_select #=> "1"
       """
 
   Scenario: selecting an option
-    Then we can change the value of the select to "Two" with:
+    Then we can change the value of the select to "2" with:
```

Comment:
This behavior doesn't seem as user-focused as we want. Why would we return the value if they selected with the text?